mailpile/plugins/setup_magic.py

```diff
@@ -577,7 +577,7 @@ def _get_xml_autoconfig(self, url, domain, email):
                             'url': docs.get('url', ''),
                             'description': str(docs.descr)
                         })
-                except AttributeError:
+                except (AttributeError, UnicodeEncodeError):
```

Comment:
Are you sure this is the right way to deal with this? The example XML you provide defines that it is encoded as UTF-8. Thus it should be processed according to this. Given that we can see the characters in your message here, they are valid UTF-8.

Reply:
You're actually right, I figured there is a better approach for this case. I'll update this pull request soon.